### PR TITLE
Support discord.js v13 in discord example

### PR DESCRIPTION
**Pull request by cxtch**
Small update to the examples/discord.js to support v13

### CODE REVIEW

File: examples/discord.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+  console.log('Usage : node discord.js <discord bot token> <channel id> <host> <port> [<name>] [<password>]');
+  process.exit(1);
 }
 
-// Load discord
-const Discord = require('discord.js')
-const client = new Discord.Client()
-let channel = process.argv[3]
+// Load discord.js
+const Discord = require('discord.js');
```
Comment (u9g) [inline examples/discord.js:16]: can you just destructure this for Intents and Client

**Comment by u9g** [comment]
run `npm run fix` to fix lint

File: examples/discord.js (hunk 1 above)
Comment (cxtch) [reply examples/discord.js:16]: Yep

**Comment by rom1504** [comment]
can someone check this works ?

**Pull request merged**